config/selectionCfg_ETau_UL16.cfg

```diff
@@ -0,0 +1,1279 @@
+OSinviso = isOS != 0 && dau1_eleMVAiso == 1 && dau2_deepTauVsJet >= 1 && dau2_deepTauVsJet < 5 # C region
+SSinviso = isOS == 0 && dau1_eleMVAiso == 1 && dau2_deepTauVsJet >= 1 && dau2_deepTauVsJet < 5 # D region
+
+baselineMcut    = baseline, massCut
+baselineInvMcut = baseline, massCutInv
+
+res1b = baseline, btagM , isBoosted != 1, massCut
+res2b = baseline, btagMM, isBoosted != 1, massCut
+
+boostedL_pnet = baseline_boosted, pNetBTagL
+boostedM_pnet = baseline_boosted, pNetBTagM
+boosted_semi  = !(pairType == 1 && nleps == 0 && (isLeptrigger || (isMETtriggerNoThresh and metnomu_et > 160)) && isBoosted == 1 && fatjet_particleNetMDJetTags_score >= 0.9137), baseline, btagLL, isBoosted==1, fatjet_hasMatchedSj==1, massCut
+boosted = boostedL_pnet || boosted_semi
+
+res1bmHH250_335   = res1b, mHH1
+res1bmHH335_475   = res1b, mHH2
+res1bmHH475_725   = res1b, mHH3
+res1bmHH725_1100  = res1b, mHH4
+res1bmHH1100_3500 = res1b, mHH5
+
+res1bmHH250_335   = res1b, mHH1
+res1bmHH335_475   = res1b, mHH2
+res1bmHH475_725   = res1b, mHH3
+res1bmHH725_1100  = res1b, mHH4
+res1bmHH1100_3500 = res1b, mHH5
+
+boostedmHH250_625	= boostedL_pnet, mHH1b
+boostedmHH625_775	= boostedL_pnet, mHH2b
+boostedmHH775_1100	= boostedL_pnet, mHH3b
+boostedmHH1100_3500 = boostedL_pnet, mHH4b
+
+#########################################################################
+#########################################################################
+
+## weights to be applied for a certain selections when calling Fill()
+## multiple weights are passed as list and are multiplied together
+## NOTE: no weight is applied for data (the simple Fill() is used)
+[selectionWeights]
+baseline		 = MC_weight, PUReweight, L1pref_weight, trigSF, IdFakeSF_deep_2d, PUjetID_SF, bTagweightReshape
+baseline_boosted = MC_weight, PUReweight, L1pref_weight, trigSF, IdFakeSF_deep_2d, PUjetID_SF, bTagweightReshape
+
+[sampleWeights]
+#TTfullyHad = TTtopPtreweight
+#TTfullyLep = TTtopPtreweight
+#TTsemiLep  = TTtopPtreweight
+#DY = DYLOtoNLOreweight
+
+# define alternative weights to be tested instead of the nominal one
+[systematics]
+#IdAndIsoAndFakeSF_deep_pt = tauid_pt20to25Up:idAndIsoAndFakeSF_tauid_pt20to25_up, tauid_pt20to25Down:idAndIsoAndFakeSF_tauid_pt20to25_down, tauid_pt25to30Up:idAndIsoAndFakeSF_tauid_pt25to30_up, tauid_pt25to30Down:idAndIsoAndFakeSF_tauid_pt25to30_down, tauid_pt30to35Up:idAndIsoAndFakeSF_tauid_pt30to35_up, tauid_pt30to35Down:idAndIsoAndFakeSF_tauid_pt30to35_down, tauid_pt35to40Up:idAndIsoAndFakeSF_tauid_pt35to40_up, tauid_pt35to40Down:idAndIsoAndFakeSF_tauid_pt35to40_down, tauid_pt40toInfUp:idAndIsoAndFakeSF_tauid_pt40toInf_up, tauid_pt40toInfDown:idAndIsoAndFakeSF_tauid_pt40toInf_down, etauFR_barrelUp:idAndIsoAndFakeSF_etauFR_barrel_up, etauFR_barrelDown:idAndIsoAndFakeSF_etauFR_barrel_down, etauFR_endcapUp:idAndIsoAndFakeSF_etauFR_endcap_up, etauFR_endcapDown:idAndIsoAndFakeSF_etauFR_endcap_down
```

Comment:
I have a preliminary version of the needed shape systematics in my configs. We can decide if we want to add it now or later.

Reply:
please, now!

Reply:
Preliminary systematics added.